sentry_sdk/utils.py

```diff
@@ -868,13 +871,24 @@ def _get_contextvars():
                 pass
         else:
             # On Python 3.7 contextvars are functional.
+            # Special check for eventlet
+            try:
+                import eventlet  # type: ignore
+                from eventlet.patcher import is_monkey_patched  # type: ignore
+                if is_monkey_patched('thread'):
+                    contextvars = eventlet.import_patched('contextvars')
```

Comment:
this line should be redundant because eventlet doesn't touch contextvars

it should work precisely as `import contextvars`

Reply:
Thanks!